lib/board.rb

```diff
@@ -0,0 +1,64 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells.clear
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(position)
+    position = position.to_i - 1
+    @cells[position]
+  end
+
+  def full?
+    @cells.include?(" ") ? false : true
```

Comment:
Use the implicit boolean return value of the expression instead of adding the ternary. You can reverse (or negate) the boolean with the `!` (bang) operator.

`!@cells.include?(" ")`